src/Directory.Build.props

```diff
@@ -19,7 +19,7 @@
     <!-- https://github.com/clairernovotny/DeterministicBuilds -->
     <ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">True</ContinuousIntegrationBuild>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
-    <NoWarn>$(NoWarn);1591</NoWarn> <!--TODO: temporary solution, remove-->
+    <NoWarn>$(NoWarn);1591;IDE1006</NoWarn> <!--TODO: temporary solution, remove-->
```

Comment:
`Async` suffix rule

Reply:
Just wondering why you decided to have the Visit methods not end in Async?

Reply:
Year or two ago I would do so, yes. I know about all those recommendations and styles. I control several hundred repos at work unifiing code style/design and this have always been one of the permanent questions. My intuition suggests me that if the API is initially designed as (completely) asynchronous, then it makes no sense to append `Async` everywhere. It only creates additional noise. By the way, I remove that suppression from dir.props, see .editorconfig file.